src/table.ts

```diff
@@ -43,19 +43,15 @@ import {CreateBackupCallback, CreateBackupResponse} from './cluster';
 import {google} from '../protos/protos';
 import {Duplex} from 'stream';
 import {TableUtils} from './utils/table';
+import {
+  DEFAULT_BACKOFF_SETTINGS,
+  RETRYABLE_STATUS_CODES,
+} from './utils/retry-options';
+import {ReadRowsResumptionStrategy} from './utils/read-rows-resumption';
 
-// See protos/google/rpc/code.proto
-// (4=DEADLINE_EXCEEDED, 8=RESOURCE_EXHAUSTED, 10=ABORTED, 14=UNAVAILABLE)
-const RETRYABLE_STATUS_CODES = new Set([4, 8, 10, 14]);
 // (1=CANCELLED)
 const IGNORED_STATUS_CODES = new Set([1]);
 
-const DEFAULT_BACKOFF_SETTINGS: BackoffSettings = {
-  initialRetryDelayMillis: 10,
-  retryDelayMultiplier: 2,
-  maxRetryDelayMillis: 60000,
-};
-
```

Comment:
`DEFAULT_BACKOFF_SETTINGS` and `RETRYABLE_STATUS_CODES` have simply been moved to `read-rows-resumption.ts`.